Exercise_1.py

```diff
@@ -2,19 +2,23 @@ class myStack:
   #Please read sample.java file before starting.
   #Kindly include Time and Space complexity at top of each file
      def __init__(self):
+       self.l = list()
          
      def isEmpty(self):
-         
+         return len(self.l) == 0
      def push(self, item):
+       self.l.append(item)
          
      def pop(self):
-        
+        return self.l.pop()
```

Comment:
What if stack underflows?

Reply:
if len(l) == 0:
     return -1

will check for this return value in main and will print("stack is empty and cannot pop from stack")